ui/desktop/app/routes/scopes/scope/projects/settings/index.js

```diff
@@ -48,6 +48,7 @@ export default class ScopesScopeProjectsSettingsIndexRoute extends Route {
 
     const logLevel = await this.ipc.invoke('getLogLevel');
     const logPath = await this.ipc.invoke('getLogPath');
+    const serverInformation = await this.ipc.invoke('getClusterUrl');
```

Comment:
What's the difference between this and using the [renderer cluster url](https://github.com/hashicorp/boundary-ui/blob/9b1f052a569fc1579befffda87de3a91ba5e417b/ui/desktop/app/services/cluster-url.js#L30) from our ember service? I don't recall how the URL works

Reply:
good point, I was not aware we have a service for cluster-url, it seems like we process the cluster url within the service here in updateClusterUrl in the application [route](https://github.com/hashicorp/boundary-ui/blob/a9d110a1b6bb9e266b21594c5f780ed1f0dbc079/ui/desktop/app/routes/application.js#L38), I'll update the route to use the url from the service